examples/siro_sandbox/controllers/controller_helper.py

```diff
@@ -101,24 +101,36 @@ def __init__(
                 )
             self.controllers.append(gui_agent_controller)
             if is_multi_agent:
-                if args.app_state == "fetch":
+                if (
```

Comment:
Instead of checking args.app_state, let's have the calling code do this logic. And then let's add some parameters to `ControllerHelper.__init__`: 1. an integer for policy_controlled_agent_index (this lets us get rid of the hacky code below for computing this index), and 2. a bool use_fetch_baselines_controller.

Background: my philosophy here is that helper classes should know as little as possible about the calling code (this makes them more reusable). Here, a tension is that it's so convenient to just pass args to this helper class rather than forward all the individual items from args that this class needs (like args.lin_speed), with the downside that access to args lets this class know a lot (basically too much) about the calling context (like the existence of app states and which app state it is being called from).

Reply:
Added use_fetch_baselines_controller argument.

Regarding policy_controlled_agent_index, agents and agent controllers are not the same things. We don't know agent controller index in advance in the calling code since agent controllers are instantiated by ControllerHelper it its __init__. 

I've added comments with explanations. Let me know if it looks good to you.